cypress/e2e/gui/engageSphere.cy.js

```diff
@@ -0,0 +1,141 @@
+    cy.getByClassThatStartsWith('ThemeToggle_button').should('be.visible');
+    cy.get('[data-testid="name"]').should('be.visible');
+  });
+
+  it('Abre e fecha o messenger', () => {
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.get('#messenger-name').should('be.visible');
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.get('#messenger-name').should('not.exist');
+  });
+
+  it('Garante que todos os campos do messenger são obrigatórios e que o primeiro está focado', () => {
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    
+    cy.get('#messenger-name').should('be.focused');
+    cy.get('#messenger-name').should('have.attr', 'required');
+    cy.get('#email').should('have.attr', 'required');
+    cy.get('#message').should('have.attr', 'required');
+  });
+
+  it('Mostra e oculta uma mensagem de sucesso ao enviar o formulário do messenger com sucesso', () => {
+    cy.clock();
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.get('#messenger-name').should('be.visible');
+    
+    cy.get('#messenger-name').type('John Doe');
+    cy.get('#email').type('john.doe@example.com');
+    cy.get('#message').type('Hello, I need support!');
+    cy.getByClassThatStartsWith('Messenger_sendButton').click();
+
+    cy.getByClassThatStartsWith('Messenger_success')
+      .should('be.visible')
+      .and('contain', 'Your message has been sent.');
+    cy.tick(3000);
+    cy.getByClassThatStartsWith('Messenger_success').should('not.exist');
+  });
+
+  it('Limpa todos os campos do formulário do messenger ao preenchê-los, fechar o messenger e abri-lo novamente', () => {
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.get('#messenger-name').should('be.visible');
+    cy.get('#messenger-name').type('John Doe');
+    cy.get('#email').type('john.doe@example.com');
+    cy.get('#message').type('Hello, I need support!');
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+
+    cy.get('#messenger-name').should('have.value', '');
+    cy.get('#email').should('have.value', '');
+    cy.get('#message').should('have.value', '');
+  });
+
+  it('Mostra as colunas Nome da Empresa e Ação, e oculta as colunas ID, Indústria, Número de Funcionários e Tamanho em um viewport móvel', () => {
+    cy.viewport(375, 667);
+    
+    cy.get('[data-testid="table"]').should('be.visible');
+    cy.contains('th', 'Company name').should('be.visible');
+    cy.contains('th', 'Action').should('be.visible');
+    cy.contains('th', 'ID').should('not.be.visible');
+    cy.contains('th', 'Industry').should('not.be.visible');
+    cy.contains('th', 'Number of employees').should('not.be.visible');
+    cy.contains('th', 'Size').should('not.be.visible');
+  });
+  
+  it('Bug - Edit name bloquado apos filtro não retornar dados', () => {
```

Comment:
A descrição do caso de teste deve ser do comportamento sendo testado, não a descrição do bug.

Algo como:

```suggestion
  it('Reativa o campo de entrada ao retornar de um filtro de estado vazio para um não vazio', () => {
```